docs/user/assets/manager-arch.drawio.svg

```diff
@@ -0,0 +1,349 @@
H/ch3BX04dFXmoFYbUuZAuYHOX8miV70LAHGdH3EbmNdGOEI9UWgZsmh+ciM0l4pyTPnQynk4tNdyR9XUZ/fCFQL7OuGC2lFLlgahAlkLcWa+M2WbdMFVtSiymZQGq/epktfX33FQDJfAC9TmEsbbQ6/Gub7x7pBg20Groj/lqt0scNpFo7OoHCtVkNZTIIXnadWPSVhzSroz7qSJHrRh7BcyjFDrmsnBPJqMoDO4hFm7bucrB6H8nxZV086BkPhfI7BVNDnAFejMbO31xDvZVfxjFz2/mmsVcKWXtEwm7OSWpq1sgoxBpp5EB0aEiHBnSJOU1n7GRFwr2BZACj/GSgyTYN5SzLltk9CF69VVMlCgnPKWw642YEDfqvssVIQryGsp3nSBbEUEAgk4NCWsWQMza946bVqNI0KfX68BglB3h3lb722z9Np7Shid47Gfl4MT2qxyJ6YHBNLPJZUXlEPZEXZDAulnpoe9q+cDVeVY0DKJMb98pgM/k5Ny9WO7pZmKk7M9tr5KrZ/FLQphwDiEN2cSVg50l3xFQyvqNgoyb05JKLArheJIm5wFX8szl8UY1cNHGoE0ZHcc3ZpubTQCsi3dOjYCLo3HpdBWhDvwRfxikCmOIXNzF/CaIXYpKdG9FMDMwL8jIpUAUC0F/IWlrr0Zz6YUIVv1levWa/npw4YDlR33xHz/NClUTfW2et505QPD3PR+9f1lp5UEszAM4wQ+7lEJIqv4MiA0HcDSxgJ83icEfXfyZLKP4MexhJcjjsbt/dD7weq3e21euzPvmBPnHKaUOy+pzfnSrybU5PBidR+uckyzVmd0O2hHAPS4kT2leuSxNUy5XJ3gooWZ9V2UOpaUBr/zlf98eNvFwtTaGjmkiiEW8oOs/CK9Hp0xf3lEHYXi+n+mwzJty3ycr2hDHhwe6AlTEhc74GjcD2th2LNw/ZBZ572UCwgzpxlLXU0nKCtK5LXhSJYtu7ar5wdHOGPN9p250M2x5D+Z7nfDJvrzcW31kt8dHykRVkYXNQO1fgm9fWCM5A9+yvrX8Yb/F2xE2zkMV2eEEY9mbss6n6bJfPLj3F9bV8bMMVmwGySffU3gh8Q8x4Cg0pZJcHXz+JokwChAbUOoropYJjHt4DL68UFSebDeOcrItRtN5NKKpGckfqeaHQ1InQ+eR0F4Z3wG4Xq+4Vtp6FZTyfygn5kxshPvlRYURKsNr9JPCCtcI44YOjzgIScnYUSJbK9PkcxaIsF8MtFMUXE026Kfi67dkmTJkqBZcNAQLWqcY1Wmcb52IcQaS+rD0ej/UpclJ/Fs0VbRRnTDMa7DKj407i6Ftz4SzQjBh7zdv38jkv5GJSNqWSlUQ7hfeTc3kgDzjwlOheN0enEvyRtI1MqmvtsZw2Nvk20jgvlohpJXZhLkSQNLEgNkhznPo4WVBcp/gsm8OESczkjV67BzR8oJ2fFfWkPIRN9/2i4PwHtXse7lMS0yKmraiHsu+7vOlf3nL5jnaG59zVusi6eyUMSINixzvKCRJ1A5UrHCvdALTm3QaBQ1Hs5DujTpO6ZKX8oor1XGB4R9HKi3VVdHlbGQhi5I1MCBI0DS/r7LSjEGEnLrdFRRIVA6rvelPwtjSPvgOwTS2ugFzTo9FB4zsueu7qYSTLLFIfDaQPpfh1//1Oj38YKRSLvNhgldoLgj1C738b6qHpWvAsMazvOF0dtCecRYogNJ3Q+1g/kTzV8/bkQw91oGgYVwPdzAZ36ZSjlnUkPIrwoocY5MOmopuv6Ip70Ls3RCNMbM35TmF3XFwbtES9la0nJWO0X4jbmuAovNSDE5F0SGGWLfnLfDDvbFBcrnK1JBDS0UpHnq1VGdfwQHoP7yEFgy0Ey3Yszr1czHahqYbVNd/irGnfzorsFGXkP9BykmTBnTH6koNlge4kBom5l5MkMA+hqlWlX7L6+HYt+SLL+fj01I+VfXMUJNTk3Rt/gimVUnPKpRwy7hrvy0WIPd6kMgxO3lXuXOiqInm4C/1dUoHXPDZHyVx6aicEi5ezLEnsw5EVEnCQ1KbLbFwN3udyXu/5PmwXRYGJ46QeyWtObp6Shvsx2gQL0ISQDLFrLq/uTFd9ebMmNJIaBIxBgWaRF41oTZUxRnOCL061HFIcaxVU/gUX3n9A4QdcDnmGY9rNEIJh6Ae0puOc7j+AcPEXnG/3c9q36TwCAox8L2Xo3xAcI3AW+bx9a+H1KSZR9Df2hxfzKd3KZC4+NSj2NxYHbBulCIqiSJT41CjSMi++dQojfqMImsVREsFRhmHZTxWggryL8987BXn2p6vQ9rLzadN87/n7O4aUybdu498GGjZL+gH9glENeCA3PaE/HDLNr+ZTQg1LDwvivunHX3AglSFjHv0nUH3BY5DvH//1nk8k67v51yxsy+b1qSqnzZrOZRz+UD6VR/opRfHn/mPB56GwpOvHNmx+KFvDsQzBZ1Pm4byM6fQ/1IvD51dVtm+TCwsJMIJ3SZPOczr+CoYfl13+51/247MIu29NYh8YQJL517JL3vgD4cj34bxL5hH8IANtfG+tSz+lYP377Y9Nbf2Y/PHhv7cF+hvVJWgOtjnNY1+nv37DoD8/EyjQ/RjOZd/9OheAhnTp9O1BZVfO5ffR/H3dH2b+p/V+QII/1MuaPpz/fphJOT2b8PW9elOCAgz5j7J99uMcwjnj/oZe4FsOP82xj0GfwfpiiN5Do8MHLwFKf1DzU+2Dv3/DUgz2FMC3opxTG8wjBG9j+ASwYm5BNwUUYvd7+vjPKIRvXR37Baxh8q1KVjbN9wq/YHgax0zGAngIMK8DsBisdjr+35APHPkDuSAY7E8EAsXIP5OE77/7X9EA4i8a8BcN+LenAfcR7OV/ORXIsgyJ6X8SFUAp8l9IBsi/yMBfZODfngxcAF6X8b8BHUiICP+nSQP/f+IAPFzs4Wz/XnYG01Nc+iSFNf4b&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
The lonely logo floating in whitespace looks weird

<img width="123" alt="image" src="https://github.com/user-attachments/assets/255a3272-19fc-451e-8068-0eafd1d58171">